.travis.yml

```diff
@@ -1,17 +1,28 @@
 language: c
 
 before_install:
+    - git submodule update --init --recursive
     - sudo add-apt-repository -y ppa:snaipewastaken/ppa
     - sudo apt-get update -q
     - sudo apt-get install criterion-dev -y
+    - git clone https://github.com/redis/hiredis.git
```

Comment:
Why is `hiredis` not a submodule?

Reply:
The API team provides hiredis as a submodule in `/api/lib/hiredis/`, but for some reason travis doesn't pull it even with `git submodule update --init --recursive`. As such, we simulate that by pulling it when we build on travis.

In the future, when it does get registered by the API team, we'll update this to 
```
    - cd api/lib/hiredis
    - make
    - sudo make install
    - sudo ldconfig
    - cd ../../..
```
but as of right now, travis doesn't recognize the files contained in there.